.github/workflows/ci.yml

```diff
@@ -390,7 +391,7 @@ jobs:
 
       ## Actual task
       - name: Run Android tests on API ${{ matrix.api-level }} / ${{ matrix.arch }} / ${{ matrix.target }}
-        uses: reactivecircus/android-emulator-runner@v2
+        uses: reactivecircus/android-emulator-runner@v2.28.0
```

Comment:
If you use `reactivecircus/android-emulator-runner@v2` CI complains and is not green